test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1052,11 +1053,11 @@ void UcxConnection::set_log_prefix(const struct sockaddr* saddr,
 
 void UcxConnection::connect_tag(UcxCallback *callback)
 {
-    const ucp_datatype_t dt_int = ucp_dt_make_contig(sizeof(uint32_t));
+    static const ucp_datatype_t dt = ucp_dt_make_contig(sizeof(uint64_t));
```

Comment:
why needed? prev constant should be compliled out, not sure if it's possible with static valiable

Reply:
removed